dimod/vartypes.py

```diff
@@ -121,7 +121,18 @@ class Vartype(enum.Enum):
 REAL = Vartype.REAL
 
 
+# when we drop 3.7 we can just use typing.Literal
 VartypeLike = typing.Union[Vartype, str, frozenset]
+"""Objects that can be interpreted as a variable type.
```

Comment:
This docstring isn't displayed in the typing.rst output HTML

Reply:
I am going to leave it alone for now - see what RTD does with it